### PR TITLE
Add tests for comments counter

### LINKED ISSUE #2: [3pt]  - Add tests for comments counter - Student B
Make sure that the function for the comments counter is covered with unit tests using Jest.

Make sure that you have tests for all edge cases.

### PR DESCRIPTION
**Pull request by Ambient567**
In this branch I add tests to the comment counter

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -197,109 +197,122 @@ footer .f-bottom {
...
-  flex-direction: column;
+.comment-sp {
+  font-style: italic;
+}
+
+.add-comment-btn:hover,
+.comment-btn:hover {
+  outline: none;
+  background: rgb(205, 158, 41);
+  color: white;
```
Comment (SSEKPIUS) [inline src/style.css]: We shall need to rearrange the classes to merge, this repo, later on before the merge is done

**Review by SSEKPIUS — APPROVED** [review]
Hi @Ambient567 , 

Good job so far!
There are a few issues that we still need to work on in order to merge, but you are almost there!


Screenshot from 2022-10-20 09-38-51



## Highlights

- Descriptive README.md file :heavy_check_mark: 
- PR has a title and summary :heavy_check_mark: 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------

**Review by rloterh — COMMENTED** [review]
Hi Team, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- [ ] You have submitted a project, that has incorrect gitflow/ Github flow. Kindly note that the gitflow of this project should have merge request flow **from development into main**. Your team members are supposed to give reviews to each other and merge their feature branches into the dev branch, which your submission **from dev branch into main**. Kindly view this article: [curriculum-transversal-skills](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/code_review_flow_group_projects.md) for further instructions on the same. 

- [ ] Kindly also ensure that all required linter checks are showing/ correctly passing on your Pull Request

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by SSEKPIUS** [comment]
> Hi Team,
> 
> While you made a great effort in this project, unfortunately, I cannot proceed to review your code.
> 
> ## Invalid Code Review Request
> * [ ]  You have submitted a project, that has incorrect gitflow/ Github flow. Kindly note that the gitflow of this project should have merge request flow **from development into main**. Your team members are supposed to give reviews to each other and merge their feature branches into the dev branch, which your submission **from dev branch into main**. Kindly view this article: [curriculum-transversal-skills](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/code_review_flow_group_projects.md) for further instructions on the same.
> * [ ]  Kindly also ensure that all required linter checks are showing/ correctly passing on your Pull Request
> 
> Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _Invalid Code Review Request **does not count** into the code reviews limit._

Thank you @rloterh , my bad indeed.
